src/noble/edwards.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2021 Cloudflare, Inc. and contributors.
+// Copyright (c) 2021 Cloudflare, Inc.
+// Licensed under the BSD-3-Clause license found in the LICENSE file or
+// at https://opensource.org/licenses/BSD-3-Clause
+
+import { Field } from '@noble/curves/abstract/modular'
+import { CHash } from '@noble/hashes/utils'
+import { CurveFn } from '@noble/curves/abstract/edwards'
+import { ElementSpec, GroupParams, ScalarHash } from './types.js'
+import { HashID } from '../cryptoTypes.js'
+
+export interface MakeEdParamsParams {
+    curve: CurveFn
+    scalarHash: ScalarHash
+    element: ElementSpec
+    hashID: HashID
+    hash?: CHash
```

Comment:
Let's be explicit in ristretto suite, and we avoid conditional execution of line 33.

```suggestion
    hash: CHash
```